test/translate_c.zig

```diff
@@ -1334,7 +1334,7 @@ pub fn addCases(cases: *tests.TranslateCContext) void {
         \\}
     ,
         \\fn ptrcast(a: [*c]c_int) [*c]f32 {
-        \\    return @ptrCast([*c]f32, a);
+        \\    return @ptrCast([*c]f32, @alignCast(@alignOf([*c]f32), a));
```

Comment:
This is not quite right, it should be `@alignOf(f32)`.

Reply:
Duh, I must've missed some type unwrapping in the last rewrite. Sorry for the noise, the patch should be ready to go now.